src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.xml

```diff
@@ -67,11 +67,11 @@
             <argument type="tagged_iterator" tag="messenger.transport_factory" />
         </service>
 
-        <service id="messenger.transport.amqp.factory" class="Symfony\Component\Messenger\Transport\AmqpExt\AmqpTransportFactory">
+        <service id="messenger.transport.amqp.factory" class="Symfony\Component\Messenger\Bridge\AmqpExt\Transport\AmqpTransportFactory">
```

Comment:
What happens when you configure an amqp transport but do not have the new bridge installed?

Line 51, still references the old `RejectRedeliveredMessageMiddleware`

Reply:
We are removing them in the FrameworkExtension